python/lsst/pipe/drivers/skyCorrection.py

```diff
@@ -175,6 +187,15 @@ def loadImage(self, cache, dataId):
         bgOld = cache.butler.get("calexpBackground", dataId, immediate=True)
         image = cache.exposure.getMaskedImage()
 
+        if self.config.doDetection:
+            # We deliberately use the specified 'detectSigma' instead of the PSF, in order to better pick up
+            # the faint wings of objects.
+            results = self.detection.detectFootprints(cache.exposure, doSmooth=True,
+                                                      sigma=self.config.detectSigma, clearMask=True)
+            if hasattr(results, "background") and results.background:
```

Comment:
Is `if results.background` this just to check if `results.background` is not None?

Reply:
Yes. I should be explicit about `is not None`?

Reply:
Actually, deliberately _not_ being explicit about `is not None`, because I'm quite happy to not trigger if the background is something like an empty list.